pkg/tsdb/request.go

```diff
@@ -2,14 +2,15 @@ package tsdb
 
 import (
 	"context"
+	"github.com/grafana/grafana/pkg/setting"
 
 	"github.com/grafana/grafana/pkg/models"
 )
 
-type HandleRequestFunc func(ctx context.Context, dsInfo *models.DataSource, req *TsdbQuery) (*Response, error)
+type HandleRequestFunc func(ctx context.Context, dsInfo *models.DataSource, req *TsdbQuery, cfg *setting.Cfg) (*Response, error)
```

Comment:
Changed the type here so all datasources needed to be changed to comply with the type.